File: JAVA/Dia07prj/src/Ejecutivo.java

public class Ejecutivo extends Empleado {
	private String bono;
	public Ejecutivo(){
		
	}
	public String getBono() {
		return bono;
	}
	public void setBono(String bono) {
		this.bono = bono;
	}

	public String desplegarInformacion() {
		
		return super.desplegarInformacion()+"\n Bono:"+this.bono+"\n\n";
	}
}

File: C#/ConsoleApplication5/ConsoleApplication5/Ejecutivo.cs
using System;

public class Ejecutivo : Empleado {
    private int id;
    private float bono;

	public Ejecutivo() {

	}

    
    public override float calcularNomina()
    {
        return ((sueldo+20)*getNumeroHorasTrabajadas())+bono;
    }

    public void setId(int id)
    {
        this.id = id;

    }

    public void setBono(float bono)
    {
        this.bono = bono;

    }

    public float getBono()
    {
        return this.bono;

    }

    public int  getId()
    {
        return id;
    }



}

